test/scripts/webdriver-update

```diff
@@ -22,7 +22,7 @@ platform_chrome='linux64'
 [ `uname` == 'Darwin' ] && platform_gecko='macos' && platform_chrome='mac64'
```

Comment:
Same issue as with the utility file. This is showing up as a new change when the repo already has it and the PR is still saying up to date even after a refresh...